src/main/java/Duke.java

```diff
@@ -6,5 +8,77 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Task[] texts = new Task[100];
+        String line;
+        String secondline;
+        int NumberOfTexts = 0;
+        int TextNumber;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("_______________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?");
+        System.out.println("\n_______________________________________________");
+
+        do{
```

Comment:
I think there is a need to put a white space between "do" and "{" to follow the coding standards. 
Similarly for the "if" and "while".